bpf/lib/overloadable_skb.h

```diff
@@ -234,6 +234,20 @@ static __always_inline bool ctx_snat_done(const struct __sk_buff *ctx)
 	return (ctx->mark & MARK_MAGIC_HOST_MASK) == MARK_MAGIC_SNAT_DONE;
 }
 
+static __always_inline void ctx_set_overlay_mark(struct __sk_buff *ctx)
+{
+	ctx->mark &= ~MARK_MAGIC_HOST_MASK;
```

Comment:
This maybe a bit confusing without further comment. 

You're just using `MARK_MAGIC_HOST_MASK` because it covers the two bits you want to clear for the overlay mark, right?

Reply:
The mark contains other information (see the big comment in `common.h`). We need to be careful about only clearing the parts that contain the "magic" (I didn't come up with the term :smile:), same as eg. `ctx_snat_done_set()`.